### ISSUE TITLE
Card reveal in horizontal card doesn't work in Firefox

### ISSUE THREAD

**Issue by aquatix** [Dogfalo/materialize#3515]
Hey, thanks for this great toolkit!

I'm currently mixing two types of cards, which works fine in Chrome, but seems to bite in Firefox: I took the new horizontal card, with an image, and added a card-content, card-reveal combo.

Any idea why the card-reveal doesn't work?

``` html
<div class="row">
    <div class="col s12 m6 l4">
        <div class="card horizontal green darken-3">
            <div class="card-image">
                <img src="" />
            </div>
            <div class="card-stacked">
                <div class="card-content white-text">
                    <span class="card-title activator">
                        title
                        <i class="material-icons right">more_vert</i>
                    </span>
                    <p>
                        content
                    </p>
                </div>
            </div>
            <div class="card-reveal green darken-3">
                <span class="card-title white-text">Card Title<i class="material-icons right">close</i></span>
                <p>revealed</p>
            </div>
        </div>
    </div>
</div>
```

**Comment by acburst** [comment]
Fixed in c88e300. Thanks for pointing this out!

**Comment by aquatix** [comment]
Cool, thanks for the quick fix, verified it works just fine now!
